src/components/ProjectConfigurationModal/ProjectConfigurationModal.js

```diff
@@ -158,28 +162,26 @@ const ProjectIconWrapper = styled.div`
 const SelectableImageWrapper = styled.div`
   display: inline-block;
   margin: 0px 10px 10px 0px;
+`;
 
-  @media ${BREAKPOINTS.sm} {
-    /* Reduce amount of displayed icons on smaller screens.
-       Todo: Check if this is really a good idea, as the currently selected icon could be hidden.
-    */
-    &:nth-of-type(n + 9) {
-      display: none;
-    }
-  }
```

Comment:
Even on small screens, the current number fits well, with the other tweaks I made :D

Reply:
Yes, removing the breakpoint is better. So there is no problem if the selected icon is at position > 9.